capstone/src/main/java/com/google/univiz/CollegeData.java

```diff
@@ -0,0 +1,50 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * CollegeData is a class representing a single university/college/institution. These parameters use
+ * data parsed from the CollegeScorecard API. This data has been converted from the ScorecardData
+ * class in order to create usable, readable accessor methods for the Univiz web application.
+ */
+@AutoValue
+abstract class CollegeData {
+
+  abstract int id();
+
+  abstract String name();
+
+  abstract String city();
+
+  /**
+   * Returns a boolean that tells whether the institution is the main campus or not, true or false.
+   */
+  abstract boolean isMainCampus();
+
+  /**
+   * Returns a string that describes the institution's degree of urbanzation, or how populated the
+   * surrounding areas are around the institution, from large city to rural.
+   */
+  abstract String urbanizationDegree();
```

Comment:
This would be better modeled as an Enum with the potential values of the urbanization degree.

Reply:
anything that has some range of values ill make sure to make an Enum.